docs/semgrep-cloud-platform/user-management.md

```diff
@@ -173,12 +181,6 @@ The following table displays typical uses for token scopes:
 | ----------- | ----------- |
 | Agent (CI)  | Generated by Semgrep Cloud Platform when onboarding (adding) a repository to Semgrep Cloud Platform. For non-GitHub-Actions users, you may have to copy and paste the token value into your CI provider's interface. |
 | Web API     | Used to access Semgrep's API. | 
-| Member      | Autogenerated by Semgrep CLI when a member is logging in through Semgrep CLI. This type of token is generated only in organizations with RBAC enabled. Use this scope to scan your code locally using your organization's private rule and rulesets. | 
-
-### Behavior of token scopes in RBAC-enabled organizations
-
-- Member-scoped tokens are only available to organizations with RBAC enabled. If RBAC is not enabled, all users are able to create and edit tokens with any scope.
-- Only the `admin` role is able to edit, view and create tokens.
-- Logging in through Semgrep CLI generates a token based on the scope of the user. Users with the `member` role create member-scoped tokens. These tokens cannot be escalated. Users with the `admin` role create agent-scoped tokens.
+| Member      | Autogenerated by Semgrep CLI when a member is logging in through Semgrep CLI. Use this scope to scan your code locally using your organization's private rule and rulesets. These tokens cannot be escalated. | 
```

Comment:
```suggestion
| Member      | Autogenerated by Semgrep CLI when a member is logging in through Semgrep CLI. Use this scope to scan your code locally using your organization's private rule and rulesets. The permissions of these tokens cannot be escalated. |
```

Thanks for including this. Just adding "permissions" to make it more explicit and in case someone ctrl+fs the page.